### ISSUE TITLE
Add more ROI class

### ISSUE THREAD

**Issue by rpeltekov** [rpeltekov/ge3t_shim_tool#10]
make more different kind of ROI that you can select.
is dependent on #1 though
currently it is only an ellipsoid, but it should be a couple more objects like 

- [ ] cube / square
- [ ] drawable manually?
- [ ] combination of square and maybe you can cut it off somehow?

Might also be worth it to in here or in another issue, add the capability for adjusting the ROI by dragging the actual display of the ROI. idk really how to do it im not that big of a gui wizard...

also: 
the tool.roiMask does not really do anything..... you should check in on that. look at gui line 895 or somewhere close to that
